api-schema/xdb.yaml

```diff
@@ -521,4 +583,23 @@ components:
       enum:
         - TERMINATE # default behaviour. ProcessStatus becomes TERMINATED
         - FAIL # ProcessStatus becomes FAILED
+    ProcessExecutionPublishToLocalQueueRequest:
+      type: object
+      description: the request for sending message(s) to be consumed within a single process execution
+      required:
+        - namespace
+        - processId
+        - queueName
+      properties:
+        namespace:
+          type: string
+        processId:
+          type: string
+        queueName:
+          type: string
+        dedupId:
```

Comment:
actually, maybe we can just let it publish an array from beginning. It doesn't seem like a lot of additional work to implement. And doing it later will require a lot of refactoring and backward compatibility work

Reply:
good idea.